components/LandingPage.tsx

```diff
@@ -25,7 +25,10 @@ const LandingPage: React.FC = () => {
         </div>
         <div className="row justify-center mt-4">
           <div className="col-md-10 offset-md-1">
-            <p style={{ lineHeight: 1.5 }} className="fw-light h4">
+            <p
+              style={{ lineHeight: 1.5, fontSize: '1.5rem' }}
```

Comment:
I think the file was already repeating styles before you work on it, but it's worth noting that it now seems to require a module CSS file so we don't have to repeat the styles for the `p` element or any other one.

If it already has one, we could create a class there with these styles and use it here.

Reply:
Good idea. I don't see a module CSS file for the landing page, but I can create one and update this PR